yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -20,7 +21,15 @@ var exportDataStatusCmd = &cobra.Command{
 
 	Run: func(cmd *cobra.Command, args []string) {
 		validateExportDirFlag()
-		err := runExportDataStatusCmd()
+		err := setUseDebeziumFlag()
+		if err != nil {
+			utils.ErrExit("setting use debezium flag: %s\n", err)
+		}
+		if useDebezium {
```

Comment:
nit: rename `setUseDebeziumFlag()` to something like `checkDebezium()` and directly use that at `line number: 28`
That function itself can exit in case of a error

Reply:
need to use flag itself now, as it requires in display function as well as we are now displaying export rows in case of dbzm as well